Automattic/woocommerce-payments
# Update Forter JS implementation

Fixes 515-gh-Automattic/woocommerce-payments-server

#### Changes proposed in this Pull Request
- Update Forter JS snippet to the latest version (v13).
- Use the new Forter TTI, that dispatch an event with the token instead of relay on cookies. Add a new rest controller with an endpoint to forward the token to the Fraud Service.
- Update Forter Service to also accept the token as parameter, keeping cookies solution as fallback.
- Disable Forter JS snippet load when the token has been sent.

#### Testing instructions
This depends on the server branch `add/forter-integration`. Be sure to configure the server as mentioned in 1054-gh-Automattic/woocommerce-payments-server. And clear the cache to get Forter included in account `fraud_services`.
- You should see the network request  `.../fraud/forter_token` with the token as payload, on any WC Pay page.
- Check that Forter Service `send_forter_token` is called with both cookie and JS event.
- JS snippet should not load if you set the option `wcpay_forter_token_sent` to your stripe account id.

-------------------
